# French translations for PACKAGE package.
# Copyright (C) 2010 THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# Vigilo language team <contact.vigilo@c-s.fr>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version:  vigilo.connector-metro\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2010-11-19 12:03+0100\n"
"PO-Revision-Date: 2015-04-15 10:54+0200\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: fr <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n > 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/connector_metro/__init__.py:45
msgid ""
"Please set the path to the configuration database generated by VigiConf "
"in the settings.ini."
msgstr ""
"Veuillez indiquer l'emplacement de la base de configuration générée par "
"VigiConf dans le fichier settings.ini."

#: src/vigilo/connector_metro/bustorrdtool.py:89
#, python-format
msgid "'%(msgtype)s' is not a valid message type for metrology"
msgstr "'%(msgtype)s' n'est pas un type de message valide pour la métrologie"

#: src/vigilo/connector_metro/bustorrdtool.py:97
#, python-format
msgid "Not a valid performance message (missing '%(tag)s' tag)"
msgstr "Message de performance non valide (il manque la balise '%(tag)s')"

#: src/vigilo/connector_metro/bustorrdtool.py:110
#, python-format
msgid "Invalid metrology value for datasource %(ds)s on host %(host)s: %(value)s"
msgstr ""
"Valeur de métrologie non valide pour l'indicateur %(ds)s de l'hôte "
"%(host)s : %(value)s"

#: src/vigilo/connector_metro/bustorrdtool.py:122
#, python-format
msgid "Skipping perf update for host %s"
msgstr "Mise à jour de la donnée de performance sur l'hôte %s ignorée"

#: src/vigilo/connector_metro/bustorrdtool.py:164
#, python-format
msgid "RRDtool could not update the file %(filename)s. Message: %(msg)s"
msgstr ""
"RRDtool n'a pas pu mettre à jour le fichier %(filename)s. Message : "
"%(msg)s"

#: src/vigilo/connector_metro/rrdtool.py:74
#, python-format
msgid "Error in rrdtool output (%(filename)s): %(output)s"
msgstr "Erreur dans la sortie de rrdtool (%(filename)s) : %(output)s"

#: src/vigilo/connector_metro/rrdtool.py:159
#, python-format
msgid "Host '%(host)s' with datasource '%(ds)s' not found in the configuration"
msgstr ""
"L'hôte '%(host)s' avec la source de données '%(ds)s' n'a pas été trouvé "
"dans le fichier de configuration !"

#: src/vigilo/connector_metro/rrdtool.py:180
#, python-format
msgid "RRDtool could not create the file: %(filename)s. Message: %(msg)s"
msgstr "RRDtool n'a pas pu créer le fichier %(filename)s. Message : %(msg)s"

#: src/vigilo/connector_metro/rrdtool.py:295
#, python-format
msgid "Unable to create the directory '%s'"
msgstr "Impossible de créer le dossier '%s'"

#: src/vigilo/connector_metro/rrdtool.py:332
#, python-format
msgid "Unable to start \"%(rrdtool)s\". Make sure the path is correct."
msgstr ""
"Impossible de lancer \"%(rrdtool)s\". Assurez-vous que le chemin est "
"correct."

#: src/vigilo/connector_metro/rrdtool.py:335
#, python-format
msgid ""
"Unable to start \"%(rrdtool)s\". Make sure RRDtool is installed and you "
"have permissions to use it."
msgstr ""
"Impossible de démarrer \"%(rrdtool)s\". Assurez-vous que RRDtool est "
"installé et que vous disposez des permissions pour l'utiliser."

#: src/vigilo/connector_metro/rrdtool.py:349
#, python-format
msgid "The RRD directory does not exist: %s"
msgstr "Le dossier RRD n'existe pas : %s"

#: src/vigilo/connector_metro/rrdtool.py:353
#, python-format
msgid "Unable to write in the directory '%(dir)s'"
msgstr "Impossible d'écrire dans le dossier '%(dir)s'"

#: src/vigilo/connector_metro/rrdtool.py:412
#, python-format
msgid "Started RRDtool subprocess: pid %(pid)d"
msgstr "Sous-processus de RRDtool démarré : pid %(pid)d"

#: src/vigilo/connector_metro/rrdtool.py:434
msgid "The process has not yet completed the previous job"
msgstr "Le processus n'a pas encore fini le job précédent"

#: src/vigilo/connector_metro/rrdtool.py:472
msgid "No deferred available in _handle_result(), this should not happen"
msgstr "Aucun deferred disponible dans _handle_result(), ça ne devrait pas arriver"

#: src/vigilo/connector_metro/rrdtool.py:499
msgid "The RRDTool process exited normally"
msgstr "Le processus RRDTool a quitté normalement"

#: src/vigilo/connector_metro/rrdtool.py:502
#, python-format
msgid ""
"The RRDTool process was terminated abnormally with exit code %(rcode)s "
"and message: %(msg)s"
msgstr ""
"Le processus RRDTool s'est terminé anormalement avec le code %(rcode)s et"
" le message : %(msg)s"

#: src/vigilo/connector_metro/rrdtool.py:512
msgid "Restarting..."
msgstr "Redémarrage..."

